packages/web-components/src/components/cta-block/cta-block.ts

```diff
@@ -122,7 +125,7 @@ class C4DCTABlock extends StableSelectorMixin(C4DContentBlock) {
     });
     return html`
       <div ?hidden="${!_hasContent}" class="${classes}">
-        <div class="${prefix}--content-item-wrapper">
+        <div class="${prefix}--content-item-wrapper" part="wrapper">
```

Comment:
Since there are two `wrapper` things here, lets be more specific with this one.
```suggestion
        <div class="${prefix}--content-item-wrapper" part="content-item-wrapper">
```